sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -331,4 +331,186 @@ class ColumnExpressionSuite extends QueryTest {
     assert(schema("value").metadata === Metadata.empty)
     assert(schema("abc").metadata === metadata)
   }
+
+  def testOneToOneMathFunction[@specialized(Int, Double, Float, Long) T]
+      (c: Column => Column, f: T => T): Unit = {
+    checkAnswer(
+      doubleData.select(c('a)).orderBy('a.asc),
+      (1 to 100).map(n => Row(f((n * 0.02 - 1).asInstanceOf[T])))
```

Comment:
any reason why we chose 100? just wondering if we can reduce this number to 10 to speed things up.